python/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [21.4.0](https://www.github.com/looker-open-source/sdk-codegen/compare/looker_sdk-v21.0.0...looker_sdk-v21.4.0) (2021-03-25)
+
+
+### Features
+
+* Looker 21.4 bindings ([#529](https://www.github.com/looker-open-source/sdk-codegen/issues/529)) ([4ecaec9](https://www.github.com/looker-open-source/sdk-codegen/commit/4ecaec93d991b9d82fd3a9ce584ee6ae8810341f))
+* **python:** Add Flask app example ([92e088e](https://www.github.com/looker-open-source/sdk-codegen/commit/92e088e30f944540054b75d58614578b8fd5dd00))
```

Comment:
@bcoe I'm stumped how this commit got included in the `python` package - all the touched files are under `examples/python/` but the `python` package path is `python/` - anything jump out at you?

Reply:
verified that 
```
GET https://api.github.com/repos/looker-open-source/sdk-codegen/commits/92e088e30f944540054b75d58614578b8fd5dd00
```
only produces associated files under `examples/python/` ...

Reply:
bugfix: https://github.com/googleapis/release-please/pull/842